------------------------------------------------------------
revno: 14160
revision-id: squid3@treenet.co.nz-20170529043852-zkf91gxhaqdj0rkn
parent: squid3@treenet.co.nz-20170529043741-9chwfs5onxuip52x
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Mon 2017-05-29 16:38:52 +1200
message:
  Add OpenSSL library details to -v output
  
  This is partially to meet the OpenSSL copyright requirement that binaries
  mention when they are using the library, and partially for admin to see
  which library their Squid is using when multiple are present in the system.
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170529043852-zkf91gxhaqdj0rkn
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: c401fe3de5518102ac6a3a4dc7b121ac415c05d4
# timestamp: 2017-05-29 04:40:04 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170529043741-\
#   9chwfs5onxuip52x
# 
# Begin patch
=== modified file 'src/main.cc'
--- src/main.cc	2017-02-26 08:52:45 +0000
+++ src/main.cc	2017-05-29 04:38:52 +0000
@@ -563,6 +563,10 @@
             printf("Service Name: " SQUIDSBUFPH "\n", SQUIDSBUFPRINT(service_name));
             if (strlen(SQUID_BUILD_INFO))
                 printf("%s\n",SQUID_BUILD_INFO);
+#if USE_OPENSSL
+            printf("\nThis binary uses %s. ", SSLeay_version(SSLEAY_VERSION));
+            printf("For legal restrictions on distribution see https://www.openssl.org/source/license.html\n\n");
+#endif
             printf( "configure options: %s\n", SQUID_CONFIGURE_OPTIONS);
 
 #if USE_WIN32_SERVICE

